pkg/gencmd/options.go

```diff
@@ -0,0 +1,19 @@
+package gencmd
+
+import (
+	"github.com/izumin5210/grapi/pkg/grapicmd"
+)
+
+type Option func(*Ctx)
+
+func WithGrapiCtx(gctx *grapicmd.Ctx) Option {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function WithGrapiCtx should have comment or be unexported